src/fourierdomain/nuft/nfft_alg.jl

```diff
@@ -107,6 +107,45 @@ end
     return nothing
 end
 
+function forward(
+    config::EnzymeRules.FwdConfig,
+    func::Const{typeof(_jlnuft!)}, 
+    RT, 
+    out::Duplicated, A::Const{<:NFFTPlan},
+    b::Duplicated{<:AbstractArray{<:Real}}
+) 
+    if EnzymeRules.needs_primal(config) && EnzymeRules.needs_shadow(config)
+        if EnzymeRules.width(config) == 1
+            func.val(out.dval, A, b.dval)
+            return Duplicated(
+                func.val(out.val, A.val, b.val), 
+                out.dval     
+            )
```

Comment:
```suggestion
            func.val(out.dval, A, b.dval)
            return Duplicated(out.val, out.dval)
```
We can't put `func.val` directly since its output is guaranteed to be nothing.